robot/robot.py

```diff
@@ -23,10 +23,14 @@ def createObjects(self):
         #   Dig | 0/1
         #   2^1 | Left/Right
         #   2^0 | Front/Rear
-        self.lf_motor = wpilib.Victor(0b00)  # => 0
-        self.lr_motor = wpilib.Victor(0b01)  # => 1
-        self.rf_motor = wpilib.Victor(0b10)  # => 2
-        self.rr_motor = wpilib.Victor(0b11)  # => 3
+        #self.lf_motor = wpilib.Victor(0b00)  # => 0
```

Comment:
I don't even need to ask why you're using binary. But I will anyway. Why?

Reply:
How so?

Reply:
Ah. Neat trick

Reply:
Code is poetry, Tim!